SwiftUIDemoAppUITests/SwiftUIDemoApp.xctestplan

```diff
@@ -9,7 +9,17 @@
     }
   ],
   "defaultOptions" : {
-
+    "environmentVariableEntries" : [
+      {
+        "key" : "STREAM_VIDEO_SECRET",
+        "value" : "$(STREAM_VIDEO_SECRET)"
+      }
+    ],
+    "targetForVariableExpansion" : {
```

Comment:
What is this doing?

Reply:
`STREAM_VIDEO_SECRET`? We need to pass this env var to tests so they can mock JWT.

Reply:
no, I meant this "targetForVariableExpansion".

Reply:
Oh, this means that this variable will be passed to the `DemoApp` target through `xctestplan`

Reply:
The UI tests can interact only with the app, so if want the SDK to receive this env var, we need to pass it to the app. Unit tests can do this straight away.